go/samples/http/main.go

```diff
@@ -47,9 +58,47 @@ func middleware(next http.Handler) http.Handler {
 	})
 }
 
-func main() {
+func runForMysql() *gosql.DB {
+	connection := "root:password@/sqlcommenter_db"
```

Comment:
nit: connection data can come from docker configuration ? 
Feel free to park it for later.

Reply:
Since this is a sample application, gonna park